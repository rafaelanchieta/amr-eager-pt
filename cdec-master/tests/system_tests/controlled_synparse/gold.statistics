-lm_nodes 11
-lm_edges 18
-lm_paths 18
-lm_nodes 11
-lm_edges 18
-lm_paths 18
