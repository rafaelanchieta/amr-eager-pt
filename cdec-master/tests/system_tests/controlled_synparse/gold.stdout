0 ||| c b c d ||| Glue=3 F1bad2=1 F5=1 F6=1 F7=1 ||| 0
0 ||| c d c b e ||| Glue=1 F1bad2=1 F4=1 F5=1 F7=1 ||| 0
0 ||| c b c d ||| Glue=2 F1bad2=1 F9=1 F6=1 F7=1 ||| 0
0 ||| b d c b e ||| Glue=1 F1bad1=1 F4=1 F5=1 F7=1 ||| 0
0 ||| c b c d ||| Glue=1 F1bad2=1 F3=1 F5=1 F7=1 ||| 0
0 ||| a d c b e ||| Glue=1 F1=1 F4=1 F5=1 F7=1 ||| 0
0 ||| b b c d ||| Glue=1 F1bad1=1 F3=1 F5=1 F7=1 ||| 0
0 ||| c d c b ||| Glue=1 F1bad2=1 F2=1 F5=1 F7=1 ||| 0
0 ||| a b c d ||| Glue=1 F1=1 F3=1 F5=1 F7=1 ||| 0
0 ||| b b c d ||| Glue=3 F1bad1=1 F5=1 F6=1 F7=1 ||| 0
0 ||| b d c b ||| Glue=1 F1bad1=1 F2=1 F5=1 F7=1 ||| 0
0 ||| a d c b ||| Glue=1 F1=1 F2=1 F5=1 F7=1 ||| 0
0 ||| b b c d ||| Glue=2 F1bad1=1 F9=1 F6=1 F7=1 ||| 0
0 ||| c c b d ||| Glue=2 F1bad2=1 F8=1 F6=1 F7=1 ||| 0
0 ||| a b c d ||| Glue=2 F1=1 F9=1 F6=1 F7=1 ||| 0
0 ||| a b c d ||| Glue=3 F1=1 F5=1 F6=1 F7=1 ||| 0
0 ||| b c b d ||| Glue=2 F1bad1=1 F8=1 F6=1 F7=1 ||| 0
0 ||| a c b d ||| Glue=2 F1=1 F8=1 F6=1 F7=1 ||| 0
1 ||| c b c d ||| Glue=3 F1bad2=1 F5=1 F6=1 F7=1 ||| 0
1 ||| c d c b e ||| Glue=1 F1bad2=1 F4=1 F5=1 F7=1 ||| 0
1 ||| c b c d ||| Glue=2 F1bad2=1 F9=1 F6=1 F7=1 ||| 0
1 ||| b d c b e ||| Glue=1 F1bad1=1 F4=1 F5=1 F7=1 ||| 0
1 ||| c b c d ||| Glue=1 F1bad2=1 F3=1 F5=1 F7=1 ||| 0
1 ||| a d c b e ||| Glue=1 F1=1 F4=1 F5=1 F7=1 ||| 0
1 ||| b b c d ||| Glue=1 F1bad1=1 F3=1 F5=1 F7=1 ||| 0
1 ||| c d c b ||| Glue=1 F1bad2=1 F2=1 F5=1 F7=1 ||| 0
1 ||| a b c d ||| Glue=1 F1=1 F3=1 F5=1 F7=1 ||| 0
1 ||| b b c d ||| Glue=3 F1bad1=1 F5=1 F6=1 F7=1 ||| 0
1 ||| b d c b ||| Glue=1 F1bad1=1 F2=1 F5=1 F7=1 ||| 0
1 ||| a d c b ||| Glue=1 F1=1 F2=1 F5=1 F7=1 ||| 0
1 ||| b b c d ||| Glue=2 F1bad1=1 F9=1 F6=1 F7=1 ||| 0
1 ||| c c b d ||| Glue=2 F1bad2=1 F8=1 F6=1 F7=1 ||| 0
1 ||| a b c d ||| Glue=2 F1=1 F9=1 F6=1 F7=1 ||| 0
1 ||| a b c d ||| Glue=3 F1=1 F5=1 F6=1 F7=1 ||| 0
1 ||| b c b d ||| Glue=2 F1bad1=1 F8=1 F6=1 F7=1 ||| 0
1 ||| a c b d ||| Glue=2 F1=1 F8=1 F6=1 F7=1 ||| 0
